test/plan.md

```diff
@@ -2,4 +2,13 @@
 
 1. a
 2. b
-3. c
\ No newline at end of file
+3. c
+
+## Notes
+
+At least three levels of testing...
+
+1. can the BB be deployed via docker-compose?
+2. can the BB interact with the IM?
+3. can an adaptor be deployed alongside it to test API compliance?
```

Comment:
I would have expected a question about whether the BB interacts with the other BBs as according to the govstack specification